tests/from.bats

```diff
@@ -345,3 +346,24 @@ load helpers
   run_buildah --log-level=error containers -f id=${cid}
   buildah rm ${cid}
 }
+
+@test "from with/without pull test" {
+  run buildah from --pull=false docker.io/nginx
+  [ $status -ne 0 ]
```

Comment:
Remove. Instead, use the status-code form of `run_buildah`:
```bash
    run_buildah 1 from ....   (or whatever the expected exit status is, if not 1)
```

Reply:
Will update it, thanks a lot for the tips.